src/lib/profiles/data-management/Current/TraitData.h

```diff
@@ -987,19 +987,14 @@ class TraitUpdatableDataSource : public TraitDataSource, protected TraitSchemaEn
     {
         kEventChangeFirst = kEvent_DataSourceMax,
 
-        /* Signals the beginning of a change record which in certain scenarios can span multiple data elements over multiple
-         * update requests (the latter only a possibility if the data being transmitted is unable to fit within a single packet)
-         */
-        kEventChangeBegin,
-
         /* Start of a data element */
         kEventDataElementBegin,
 
         /* End of a data element */
         kEventDataElementEnd,
 
-        /* End of a change record */
-        kEventChangeEnd,
+        /* a change record happend */
+        kEventChangeHappend,
```

Comment:
This name is not specific enough.  As far as I can tell, this event is raised at the end of update processing if the data source was updated.  Thus the name should be something like UpdateProcessingComplete.

Reply:
Done, thanks